File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        index1=0
        while True:
            for index2 in range(index1+1,len(nums)):
                if nums[index1] == target - nums[index2]:
                    return [index1,index2]       
            index1+=1
                   
        

File: 0001-two-sum/0001-two-sum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    int* result = (int*)malloc(2 * sizeof(int));
    for (int i = 0; i < numsSize; i++) {
        for (int j = i + 1; j < numsSize; j++) {
            if ((nums[i] + nums[j]) == target) {
                result[0] = i;
                result[1] = j;
                returnSize[0] = 2;
                return result;
            }
        }
    }
    return NULL; 
}
